metricbeat/module/windows/services/services_windows.go

```diff
@@ -322,9 +322,10 @@ func (reader *ServiceReader) Read() ([]common.MapStr, error) {
 			"service_name": service.ServiceName,
 			"state":        service.CurrentState,
 			"start_type":   service.StartType,
-			"uptime": map[string]interface{}{
-				"ms": service.Uptime,
-			},
+		}
+
+		if service.Uptime > 0 {
+			ev.Put("uptime", map[string]interface{}{"ms": service.Uptime})
```

Comment:
Use `ev.Put("uptime.ms", service.Uptime)`. It will automatically do the right thing.